.github/workflows/deploy-registry.yaml

```diff
@@ -0,0 +1,25 @@
+name: deploy-registry
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
```

Comment:
Do we want to deploy on each PR too?

Reply:
No I should pull that out, was moving too quickly.